### ISSUE TITLE
create couchdb 2.0 rpm

### ISSUE THREAD

**Issue by ozbarshalom** [wendall911/couchdb-rpm#24]
new rpm for version 2.0 is needed

**Comment by wendall911** [comment]
I agree! I'll get to this soon. I've been busy and there is quite a bit of work to be done around this.

**Comment by wohali** [comment]
CouchDB has official 2.x RPMs now! Download details are here:

http://docs.couchdb.org/en/2.1.1/install/unix.html#installation-using-the-apache-couchdb-convenience-binary-packages
